plugins/services/src/js/pages/EditFrameworkConfiguration.js

```diff
@@ -0,0 +1,131 @@
+import React, { PropTypes } from "react";
+import mixin from "reactjs-mixin";
+import { StoreMixin } from "mesosphere-shared-reactjs";
+import { routerShape } from "react-router";
+import deepEqual from "deep-equal";
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+import UniversePackage from "#SRC/js/structs/UniversePackage";
+import FrameworkConfiguration from "#SRC/js/components/FrameworkConfiguration";
+import Loader from "#SRC/js/components/Loader";
+
+export default class EditFrameworkConfiguration extends mixin(StoreMixin) {
```

Comment:
shouldn't we supposed to be using `module.exports` instead `export default` if I'm not wrong we might have issues related to jest and mocking.

Reply:
Using `export default` is fine here, but there are a few cases in which we have to use `module.exports` due to issue with the current plugin implementation.